utils/binutils/logger.go

```diff
@@ -0,0 +1,134 @@
+package binutils
```

Comment:
We already have `application/logger.go`.

Reply:
Ugh, thanks for catching that. I thought I removed the `binutils` package when I rebased.